src/blockserver/backend/quota.py

```diff
@@ -0,0 +1,21 @@
+
+class QuotaPolicy:
+
+    METAFILE_THRESHOLD = 150 * 1024
+    TRAFFIC_THRESHOLD = 100 * 1024**3
+
+    @staticmethod
+    def upload(quota_reached, file_size, is_block, is_overwrite=False):
+        if not quota_reached:
+            return True
+        if is_block:
+            return False
+        return is_overwrite and file_size < QuotaPolicy.METAFILE_THRESHOLD
```

Comment:
hint: logical grouping of file_size < QuotaPolicy.METAFILE_THRESHOLD and is_block would increase readability (is it ment to be a metadata upload and do we believe that it is really metadata)